ted_sws/notice_validator/services/shacl_test_suite_runner.py

```diff
@@ -58,14 +58,17 @@ def execute_test_suite(self) -> SHACLTestSuiteValidationReport:
                                               object_data="SHACLTestSuiteExecution")
 
 
-def generate_shacl_report(shacl_test_suite_execution: SHACLTestSuiteValidationReport) -> SHACLTestSuiteValidationReport:
+def generate_shacl_report(shacl_test_suite_execution: SHACLTestSuiteValidationReport,
+                          notice_id: List[str] = None) -> SHACLTestSuiteValidationReport:
     """
         This function generate html report after SHACL test execution.
+    :param notice_id:
     :param shacl_test_suite_execution:
     :return:
     """
-    html_report = TEMPLATES.get_template(SHACL_TEST_SUITE_EXECUTION_HTML_REPORT_TEMPLATE).render(
-        shacl_test_suite_execution.dict())
+    tpl_data: dict = shacl_test_suite_execution.dict()
```

Comment:
Rename tpl with template, be more explicitly.

Reply:
ok. tpl is the same as template if you want to save some bytes. in web programming (that uses templates a lot) tpl is prioritised ( a web programmer sees "template" when "tpl" is found). sincerly, it's just a convension, so I will not object.